app/assets/javascripts/browsered/web-payments/helpers.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
-const allowedCardTypes = window.Card
-const { email_collection_mode } = window.Charge
+const allowedCardTypes = window.Card || {}
+const { email_collection_mode } = window.Charge || {}
```

Comment:
This line will still result in `undefined` if window.Charge doesn't exit - I don't think this is meaningful